Add missing props mediaPreview in the ReadMe (#38114)
WordPress/gutenberg: packages/block-editor/src/components/media-placeholder/README.md

-   Default: `false`
-   Platform: Web

### mediaPreview

> React Element to render as preview in placeholder

Reviewer: ```suggestion
The component is rendered as a preview in the placeholder.
```

> -   Type: `Object`

Reviewer: ```suggestion
-   Type: `Component`
```
